content/webapp/types/manifest.ts

```diff
@@ -5,20 +5,32 @@ import {
   AuthClickThroughService,
   AuthAccessTokenService,
   Canvas,
+  InternationalString,
+  SpecificationBehaviors,
+  ContentResource,
+  ResourceType,
 } from '@iiif/presentation-3';
 import { Audio, Video } from '@weco/content/services/iiif/types/manifest/v3';
 
 export type ThumbnailImage = { url: string; width: number };
 
+export type BornDigitalData = {
+  originalFile: string | undefined;
+  label: InternationalString | null | undefined;
+  format?: string;
```

Comment:
np; the use of `undefined` version `?` in this case should it be aligned as I think a value always(?) gets passed in?